hack/update-checksum-windows.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+WORK_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+YAMLS_DIR="${WORK_DIR}"/../build/yamls
+MANIFESTS=$(ls $YAMLS_DIR/antrea-windows*.yml)
+WINDOWS_DIR="${YAMLS_DIR}"/windows
+
+checksum_windows_config=$(cat ${WINDOWS_DIR}/base/conf/* | sha256sum | cut -d " " -f 1)
+
+checksum_default=$(cat ${WINDOWS_DIR}/default/conf/* | sha256sum | cut -d " " -f 1)
+
+checksum_containerd=$(cat ${WINDOWS_DIR}/containerd/conf/* | sha256sum | cut -d " " -f 1)
+
+checksum_containerd_with_ovs=$(cat ${WINDOWS_DIR}/containerd/conf/* \
+    ${WINDOWS_DIR}/containerd-with-ovs/conf/* | sha256sum | cut -d " " -f 1)
+
+for file in ${MANIFESTS[@]}; do
+    sed -i.bak "s/windows-config-checksum-placeholder/${checksum_windows_config}/g" ${file}
+done
+
+if [ -f "${YAMLS_DIR}/antrea-windows.yml" ]; then
+    sed -i.bak "s/agent-windows-checksum-placeholder/${checksum_default}/g" ${YAMLS_DIR}/antrea-windows.yml
+fi
+if [ -f "${YAMLS_DIR}/antrea-windows-containerd.yml" ]; then
+    sed -i.bak "s/agent-windows-checksum-placeholder/${checksum_containerd}/g" ${YAMLS_DIR}/antrea-windows-containerd.yml
+fi
+if [ -f "${YAMLS_DIR}/antrea-windows-containerd-with-ovs.yml" ]; then
+    sed -i.bak "s/agent-windows-checksum-placeholder/${checksum_containerd_with_ovs}/g" ${YAMLS_DIR}/antrea-windows-containerd-with-ovs.yml
+fi
```

Comment:
what's the purpose of checking if the file exits first?

Reply:
removed as set line is added now.